src/UsgsAstroLsSensorModel.cpp

```diff
@@ -1862,16 +1789,16 @@ void UsgsAstroLsSensorModel::losToEcf(
    double fractionalLine = line - floor(line);
 
    // Compute distorted image coordinates in mm (sample, line on image (pixels) -> focal plane
-   double natFocalPlaneX, natFocalPlaneY;
-   computeDistortedFocalPlaneCoordinates(fractionalLine, sampleUSGSFull, natFocalPlaneX, natFocalPlaneY);
+   std::tuple<double, double> natFocalPlane; //double natFocalPlaneX, natFocalPlaneY;
+   natFocalPlane = computeDistortedFocalPlaneCoordinates(fractionalLine, sampleUSGSFull, m_detectorSampleOrigin, m_detectorLineOrigin, m_detectorSampleSumming, m_startingSample, m_detectorLineOffset, m_iTransS, m_iTransL);
 
    // Remove lens distortion
    double focalPlaneX, focalPlaneY;
-   computeUndistortedFocalPlaneCoordinates(natFocalPlaneX, natFocalPlaneY, focalPlaneX, focalPlaneY);
+   computeUndistortedFocalPlaneCoordinates(std::get<0>(natFocalPlane), std::get<1>(natFocalPlane), focalPlaneX, focalPlaneY);

```

Comment:
Is this input, output parameter scheme something we want to keep around?

Reply:
I'd like to remove it in favor of just returning an object.

Reply:
@jessemapel @acpaquette Sounds good to me. I think this particular function is one that is removed or replaced in @acpaquette's PR #172? Does the same general feedback apply to everything else in here?

Reply:
I think so, should we do that now, or wait until #172 is done?